integration/util.go

```diff
@@ -30,10 +30,12 @@ import (
 	"github.com/sirupsen/logrus"
 	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
+	extensionsv1beta1 "k8s.io/api/extensions/v1beta1"
```

Comment:
something feels weird to me about using `k8s.io/api/extensions/v1beta1` and `k8s.io/client-go/kubernetes/types/extensions/v1beta1` together. do these not share a common interface that we could make use of here?